src/main/java/uk/gov/hmcts/ccd/domain/model/refdata/BuildingLocation.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.hmcts.ccd.domain.model.refdata;
+
+import com.fasterxml.jackson.databind.PropertyNamingStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import lombok.Builder;
+import lombok.Data;
+import lombok.Value;
+import lombok.extern.jackson.Jacksonized;
+
+import java.io.Serializable;
+import java.util.List;
+
+@Data
+@Value
+@Builder
+@Jacksonized
+@JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy.class)
+public class BuildingLocation implements Serializable {
```

Comment:
do we need `implements Serializable` still ?

Reply:
It seems so... I get errors without it